sdk/servicebus/service-bus/test/stress/stressTestsBase.ts

```diff
@@ -360,105 +365,92 @@ export class SBStressTestsBase {
   public async snapshot(): Promise<void> {
     // TODO: get the options being set in the logs
     // TODO: Get a title passed from the scenario file
```

Comment:
I think this is done now?

Reply:
Yes, it is!

Reply:
(both of them, since I also dump the options into the tracked event at the start of the run)